content/zmc-challenges/first-website/hosting-your-page-on-github-pages/_index.md

```diff
@@ -56,61 +58,67 @@ From now on you'll be able to interact with your repo by navigating to it in you
 
-You'll be asked if you want to add your code to the main branch or if you want to create a new branch. Branches are cool, trust me, but we aren't going to get into them right now.  Just choose "main branch" 
+You'll be asked if you want to add your code to the main branch or if you want to create a new branch. Branches are cool, trust me, but we aren't going to get into them right now. Just choose "main branch"
 
 Once you have filled in the form you can save your changes by clicking on the "commit" button.
 
 ### 4. Check that you uploaded your file correctly
 
-Navigate to your repo and make sure you can see a file named `index.html`. Make sure that the contents are what they should be.  You can click on the file to see what is inside.
+Navigate to your repo and make sure you can see a file named `index.html`. Make sure that the contents are what they should be. You can click on the file to see what is inside.
 
-### 5. Set up GitHub pages 
+### 5. Set up GitHub pages
 
-You'll now use Github pages to host your website on the web. 
+You'll now use Github pages to host your website on the web.
 
-Github pages are a really powerful thing and they can be configured to do all kinds of cool stuff. We're going to use the simplest setup possible :) 
+Github pages are a really powerful thing and they can be configured to do all kinds of cool stuff. We're going to use the simplest setup possible :)
 
 [This guide](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site) will take you through the process. You can also use the following [video demonstration](https://www.youtube.com/watch?v=74v58saH8dE) for reference.
 
 You'll be asked to choose a publishing source. You'll just use the main branch and the root directory.
 
-### 6. View your live website 
+### 6. View your live website
 
 Once you have set your website up you'll be able to visit it in your browser.
 
 If your repo is available at `https://github.com/mysteryElephant320/first-website`, then your webpage will be available at `https://mysteryElephant320.github.io/first-website/`.
 
-Navigate to your website. Does it look the way you expected? 
+Navigate to your website. Does it look the way you expected?
 
 **Note:** Sometimes it can take a few minutes for the website to go live. If it's not working then wait a few minutes and check again.
 
-### 7. Submit your work 
+We have also created some video demos to walk you through some of the steps above:
+
+- [PC/Computer demo](https://www.youtube.com/watch?v=auWR5JOiZt4)
+- [Android phone demo](https://drive.google.com/file/d/1yBKPaq0yjMIqZxEgPSTgNdlkGTsswjqh/view?usp=sharing)
```

Comment:
Good job. 
I am not sure about the location you chose to store this video on. Every other video demo, including those made by sheena, have been recorded with Streamyard and made available on the Umuzi YouTube channel